src/AppSystem/Launcher.vala

```diff
@@ -328,4 +353,24 @@ public class Dock.Launcher : BaseItem {
             multiple_windows_open = app.windows.length > 1;
         }
     }
+
+    private void on_enter (Gtk.EventControllerMotion controller, double event_x, double event_y) {
```

Comment:
If you don't use the object in signal handler, you can just remove it :)

```suggestion
    private void on_enter (double event_x, double event_y) {
```

Reply:
Good to know! Removed all of them.